docs/reference/sync-configuration.md

```diff
@@ -38,6 +38,7 @@ Alternatively, these configurations can be passed to flagd via config file, spec
 | providerID  | optional `string`  | Value binds to grpc connection's providerID field. gRPC server implementations may use this to identify connecting flagd instance                                                                                |
 | selector    | optional `string`  | Value binds to grpc connection's selector field. gRPC server implementations may use this to filter flag configurations                                                                                          |
 | certPath    | optional `string`  | Used for grpcs sync when TLS certificate is needed. If not provided, system certificates will be used for TLS connection                                                                                         |
+| maxMsgSize  | optional `int`     | Used for grpcs sync to set max receive message size (in bytes) e.g. 5242880 for 5MB. If not provided, used the deafult [4MB](https://pkg.go.dev/google.golang.org#grpc#MaxCallRecvMsgSize)                       |
```

Comment:
QQ: should we make the input more user friendly e.g.  5 for 5mb or static enum

Reply:
`MaxCallRecvMsgSize` accepts an int mapping to bytes, so we should keep that format. So users can use a simple calculation `5 * 1024* 1024` to derive their value.